protobuf/pubspec.yaml

```diff
@@ -9,6 +9,7 @@ environment:
   sdk: '>=2.0.0-dev.17.0 <3.0.0'
 dependencies:
   fixnum: '>=0.9.0 <0.11.0'
+  quiver: ^2.0.2
```

Comment:
I'd prefer we do not get further dependencies if we can avoid it. I think it is better to duplicate it.
Hopefully we can have a resolution of https://github.com/dart-lang/sdk/issues/11617 someday, and use core library functionality.